CMP1903_A1_2324/Program.cs

```diff
@@ -1 +1,55 @@
-Empty
+using System;
+
+namespace CMP1903_A1_2324
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * Create a Game object and call its methods.
+             * Create a Testing object to verify the output and operation of the other classes.
+             */
+
+            //Instantiating a new Game object
+            Game game = new Game();
+            while (true)
+            {
+                Console.WriteLine("Roll dice? (yes/no)");
+                string answer = Console.ReadLine().ToLower(); //Converts all inputs to lower case
+                if (answer.Length < 1) //Exception Handling - Guard Clause to catch empty strings
+                {
+                    Console.WriteLine("Please input a string");
+                    continue;
+                }
+                else if (answer == "yes")
+                {
+                    game.CreateDice();
+                }
+                else if (answer == "no")
+                {
+                    break;
+                }
+                else
+                {
+                    Console.WriteLine("Please input either 'yes' or 'no'");
+                }
+            }
+            game.PrintDice();
+            game.CalcSum();
+            Console.WriteLine($"The sum of all of the dice rolls is {game.PrintSum()}");
```

Comment:
Here you have printed the Sum of the dice rolls effectively and clearly, also the wording makes it clear that this is where all the sums are going to happen. 
However, as stated in the requirements this was meant to be included within the game class, with this you could use the same line of code practically but you could just use sum instead of the PrintSum() method to make it easier.